README.md

```diff
@@ -165,6 +165,11 @@ Requests can then be made with post requests, for example:
 
     curl --header "Content-Type: application/json" --request POST --data '{"url":"<YOUR_URL_HERE>"}' http://localhost:8080/analyze
 
+For more accurate responses, you can use `/analyze/staged` endpoint. This will submit snippets to model for individual analysis first
```

Comment:
```suggestion
For more accurate responses, you can use `/analyze/staged` endpoint. This will submit snippets to model for individual analysis first.
```